hail/python/hail/backend/backend.py

```diff
@@ -43,6 +45,20 @@ def matrix_read_type(self, mir):
         jir = self._to_java_ir(mir)
         return tmatrix._from_java(jir.typ())
 
+    def from_spark(self, df, key):
+        return Table._from_java(Env.hail().table.Table.fromDF(Env.hc()._jhc, df._jdf, key))
+
+    def to_spark(self, t, flatten):
```

Comment:
It seems this is the case, but to check, we don't need `flatten = True` in the signature, because the current calls that need to be flattened by default only come from Table, which retains the `flatten` default?

Reply:
Correct!